<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionVoyage</class>
 <widget class="QMainWindow" name="gestionVoyage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tayarni - Gestion Voyage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Outfit</family>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="font">
      <font>
       <family>Outfit</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="Information_Box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>371</width>
        <height>661</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Outfit</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Voyage</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Flight Number</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Place de depart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Place d'arrivee</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date de depart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date d'arrivee</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Airline</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre de place valables dans l'avion</string>
       </property>
      </widget>
      <widget class="MyButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>590</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="MyButton" name="pushButton_clear">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>590</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_flightref">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>140</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>FlightRef</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_montant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="placeholderText">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nbper">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>Nombre des places</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_lieudep">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>73</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_lieuarr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>73</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="dateEdit_datedep">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit *
{
background-color:rgba(5, 24, 150);
}</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Airline">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>73</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisair</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Emirates Airlines</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="dateEdit_datearr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="displayFormat">
        <string>dd-MM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="MyButton" name="pushButton_Search">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="MyButton" name="pushButton_update">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>590</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rfid">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>520</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>470</y>
        <width>491</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Cout moyen</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cout Moyen ≈</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_lieu_From">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_lieu_To">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_moyen">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>56</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>460</width>
        <height>221</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>700</y>
        <width>491</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Profit</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Profit ≈</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>From</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>To</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_lieu_From_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_lieu_To_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>73</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Tunisia</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>France</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>United Kingdom</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_profit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>120</y>
         <width>56</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>861</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>859</width>
         <height>359</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>271</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lineEdit_rechercher">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>220</y>
         <width>110</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>110</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/icons8-pdf-48.png</normaloff>../../../../Downloads/icons8-pdf-48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_sort">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>110</width>
         <height>28</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>FLIGHTREF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LIEUDEP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LIEUARR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATEDEP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATEARR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AIRLINE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MONTANT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NBPER</string>
        </property>
       </item>
      </widget>
      <widget class="MyButton" name="pushButton_delete">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>90</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_delete">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>110</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>90</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>90</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Export to Pdf:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>90</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>710</y>
        <width>870</width>
        <height>200</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
